# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rsync/pkg_rsync.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

One type fix and the gettimeofday misdetection - the later could need some
in depth review.

  - Rene Rebe <reneÂ@exactcode.de>

diff -ur rsync-2.6.3pre2/lib/compat.c rsync-2.6.3pre2-diet/lib/compat.c
--- rsync-2.6.3pre2/lib/compat.c	2004-01-20 01:59:26.000000000 +0100
+++ rsync-2.6.3pre2-diet/lib/compat.c	2005-02-03 00:14:02.275505872 +0100
@@ -196,9 +196,5 @@
 /* some systems don't take the 2nd argument */
 int sys_gettimeofday(struct timeval *tv)
 {
-#if HAVE_GETTIMEOFDAY_TZ
 	return gettimeofday(tv, NULL);
-#else
-	return gettimeofday(tv);
-#endif
 }
diff -ur rsync-2.6.3pre2/rsync.h rsync-2.6.3pre2-diet/rsync.h
--- rsync-2.6.3pre2/rsync.h	2004-08-03 17:41:17.000000000 +0200
+++ rsync-2.6.3pre2-diet/rsync.h	2005-02-03 00:13:07.860778168 +0100
@@ -326,7 +326,7 @@
 #define uint64 unsigned long long
 #else
 /* As long as it gets... */
-#define uint64 unsigned off_t
+#define uint64 uint64_t
 #endif
 
 /* Starting from protocol version 26, we always use 64-bit
